newrelic/structures_newrelic_synthetics_simple_browser_monitor.go

```diff
@@ -1,13 +1,16 @@
 package newrelic
 
 import (
+	"errors"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/newrelic/newrelic-client-go/v2/pkg/synthetics"
 )
 
-func buildSyntheticsSimpleBrowserMonitor(d *schema.ResourceData) synthetics.SyntheticsCreateSimpleBrowserMonitorInput {
+// TODO: Reduce the cyclomatic complexity of this func
+// nolint: gocyclo
```

Comment:
I think splitting this function into two can help reduce cyclomatic complexity of this function, for instance, we can shift all of the logic that deals with runtime values and device emulation into a new function. Can you give this a try and see if it works? If not, we can then consider this temporary workaround until we find a better solution